### DISCUSSION TITLE
Ajouter des histoires plus diverses au doNothing

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#1362] [Idées acceptées / déjà implémentées]
Juste du full texte qui raconte une histoire, qui serait canon avec le reste du bot et incluse dans le lore du coup, sur un peu tous les thèmes (sauf la mort :pQuack:)

Si il en existe déjà des proposés mettez les dans les commentaires

**Comment by Stitch-seagull** [comment]
Apparemment perte de temps mais c'est plutôt un doNothing
https://github.com/DraftBot-A-Discord-Adventure/Events/discussions/256

**Comment by Stitch-seagull** [comment]
https://github.com/DraftBot-A-Discord-Adventure/DraftBot/discussions/930

**Comment by Stitch-seagull** [comment]
https://github.com/DraftBot-A-Discord-Adventure/DraftBot/discussions/711

**Comment by Stitch-seagull** [comment]
https://github.com/DraftBot-A-Discord-Adventure/DraftBot/discussions/714
